### ISSUE TITLE
Rename repo to match package?

### ISSUE THREAD

**Issue by hadley** [seankross/datamations#7]
Or vice-versa — having one name to remember is easy than two.

**Comment by seankross** [comment]
@xiaoyingpu Do you think it would be a good idea to have the name of the package match the name of the paper? (Which would be "datamations").

**Comment by xiaoyingpu** [comment]
Yeah I had it as plural earlier, but then I saw `library(animation)`, so I thought singular would make more sense in the code...

**Comment by seankross** [comment]
My thought was that if all of the names were consistent then it would be easier for people to cite us 😁. Also I liked your blurb about the paper in the readme you should bring that back!
